src/rprblender/ui/render.py

```diff
@@ -174,7 +174,14 @@ def draw(self, context):
         col.prop(settings, 'use_gl_interop')
 
         col.prop(rpr, 'viewport_denoiser')
-        col.prop(rpr, 'viewport_upscale')
+        col1 = col.column()
+
+        col1.prop(rpr, 'viewport_upscale')
```

Comment:
Disable viewport_upscale_quality if viewport_denoiser is unset.

Reply:
fixed https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/pull/577/commits/bdb00ef37826dc9b26876222f8cd5e316a372e6b